Acceptance-Tests.md

```diff
@@ -55,45 +67,69 @@ Docker:
 
-4) The utility files are imported in the top-level test files and the methods are called to perform the required actions. For ex. in the `create-draft-and-delete-draft-blog-post.spec.js` file, the `createDraftBlogPostWithTitle` method is called to create a draft blog post with a given title. The `deleteDraftBlogPostWithTitle` method is called to delete the draft blog post with the given title.
+5) Adding a New User-Utilities File to the User-Factory:
 
-5) For each test, the user is created using the `userFactory` class. For ex. in the `create-draft-and-delete-draft-blog-post.spec.js` file, the `createNewBlogPostEditor` method is called to create a new blog post editor user. The `createNewBlogPostEditor` method is defined in the `user-factory.js` file. The `createNewBlogPostEditor` method creates a new user with the given username and returns the user object. The user object is used to perform the required actions (that are defined in the `user-utilities/*-utils.js`).
+   If the role requires a super admin to assign it, first, add the role to the `Roles` enum in `test-constants.ts`. Then, to add it, reference the `USER_ROLE_MAPPING` inside the `user-factory.ts` file. If the user requires a role from the super admin, add the reference accordingly.
 
-6) After successful completition of any test step or any expectation, the `showMessage` method is called to log the progress. For ex. in the `create-draft-and-delete-draft-blog-post.spec.js` file, the `showMessage` method is called to log the progress after the draft blog post is created. The `showMessage` method is defined in the `show-message-utils.js` file.
+   For example, if we want to add Translation admin with the help of super admin then:
 
-7) If there is any error during the test, then we throw errors in the expectation step or there would be timeout error if some component does not behave as intended.
+    i) Define the role in `Roles` enum:
+    ```
+      Roles: {
+        other roles... ,
+        TRANSLATION_ADMIN: 'translation admin',
+      }
+    ```
+    ii) Add the role to `USER_ROLE_MAPPING`:
+    ```
+      const USER_ROLE_MAPPING = {
+        other roles... ,
+        [ROLES.TRANSLATION_ADMIN]: TranslationAdminFactory,
+      } as const;
+    ```
 
-8) The `puppeteer-testing-utilities` directory contains all the utility files and helper functions, which you would require to write new acceptance tests. This directory can also be used to append more utility functions as when required or needed by the user.
+    For roles that don't require super admin privileges, such as `LoggedInUser`, guide the developer to add the factory to the array inside `createNewUser` under `composeUserWithRoles(BaseUserFactory(), [...])`. This ensures that the new user role is included when creating a new user instance.
```

Comment:
no guide, just say you should add the factory to the array inside ...

Reply:
done